### PR TITLE
[Card] Add Card component with nav

### PR DESCRIPTION
**Pull request by cavasinf**
## Description

image

```twig
{% set raw_html_example %}
    <div class="card-body">
        <h3><span class="badge bg-success">Active</span> content</h3>
        <p class="mb-0">With some text</p>
    </div>
    <div class="card-footer">
        <div class="d-flex">
            <a href="#" class="btn btn-link">Cancel</a>
            <a href="#" class="btn btn-primary ms-auto">Go somewhere</a>
        </div>
    </div>
{% endset %}

{% set items = [
    {
        name : 'First',
        content : 'First content',
    },
    {
        name : '<i class="fas fa-times me-2"></i>Disabled',
        content : 'Disabled content',
        disabled : true,
        raw : true,
    },
    {
        name : 'Active',
        content : raw_html_example,
        active : true,
        raw : true,
    },
    {
        name : 'Url',
        url : 'https://www.w3schools.com/',
    },
] %}

{% embed '@Tabler/embeds/card_nav.html.twig' with { items : items} %}{% endembed %}

{% embed '@Tabler/embeds/card_nav.html.twig' with { items : items, pills: true} %}
    {% block card_nav_after %}
        <li class="nav-item ms-auto">
            <a class="nav-link" href="#">
                <i class="fas fa-cog"></i>
            </a>
        </li>
    {% endblock %}
{% endembed %}

{% embed '@Tabler/embeds/card_nav.html.twig' with { items : items, tabs : true} %}{% endembed %}

{% embed '@Tabler/embeds/card_nav.html.twig' with { items : items, tabs : true, bottom : true} %}{% endembed %}

{% embed '@Tabler/embeds/card_nav.html.twig' with { items : items, tabs : true} %}
    {% block card_content %}
        <div class="card">
            <div class="card-body">
                <h2>Single page/tab content</h2>
                <p>
                    I'm a content that will not be changed.<br/>
                    Even if you click tabs!
                </p>
                <blockquote>Useful when one tab correspond to one specifics URL</blockquote>
            </div>
        </div>
    {% endblock %}
{% endembed %}
```

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist
- [x] I updated the documentation (see [here](https://github.com/kevinpapst/TablerBundle/tree/master/docs))
- [x] I agree that this code will be published under the [MIT license](https://github.com/kevinpapst/TablerBundle/blob/master/LICENSE)

### CODE REVIEW

File: docs/embeds-card-nav.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+### Parameters
+`Card navigation` embed, can be used with 3 parameters:
+
+| Parameter | Description                                             |  Type   | Default |
+|:---------:|---------------------------------------------------------|:-------:|:-------:|
+|   items   | Array of [Item](#Item)                                  | `array` |  `[]`   |
+|   tabs    | Use `card` for each tab content instead one single card | `bool`  | `false` |
+|   pills   | Use `pills` as tab style                                | `bool`  | `false` |
+
```
Comment (kevinpapst) [inline docs/embeds-card-nav.md:19]: bottom parameter is missing

File: templates/embeds/card_nav_header.html.twig [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                        href="{{ _url }}"
+                        {% if _disabled %}tabindex="-1" aria-disabled="true"{% endif %}
+                        {% if _tabs %}
+                            {% if _active %}aria-selected="true"{% endif %}
+                            role="tab"
+                        {% endif %}
+                        {% if item.url is not defined and not _manualContent %}data-bs-toggle="tab"{% endif %}
+                >
+                    {% if _raw %}
+                        {{ _name | raw }}
```
Comment (kevinpapst) [inline templates/embeds/card_nav_header.html.twig:41]: Is the only reason the icon in your example? If so, I would prefer a .icon check instead of raw.

**Review by kevinpapst — CHANGES_REQUESTED** [review]

**Comment by kevinpapst** [comment]
Looks very good 👍

File: docs/embeds-card-nav.md (hunk 1 above)
Comment (cavasinf) [reply docs/embeds-card-nav.md:19]: Nice catch 👍

File: templates/embeds/card_nav_header.html.twig (hunk 2 above)
Comment (cavasinf) [reply templates/embeds/card_nav_header.html.twig:41]: Not only, in my case I add sometimes badges/dropdowns/...
So I need to have the fully control of what I can put in there.

**Comment by cavasinf** [comment]
@kevinpapst Should I name it `card_nav.html.twig` or `card-nav.html.twig`?
I'll tend to say `card-nav.html.twig`

**Review by kevinpapst — APPROVED** [review]
Thank you @cavasinf 👍

**Comment by kevinpapst** [comment]
I updated the demo with a new page. 
Thanks for sharing the code examples, that made it much simpler :)

**Comment by kevinpapst** [comment]
Ah, overlooked the filename question. 

I would probably rename:
- `card-vertical-navigation.html.twig` to `card_navigation_vertical.html.twig`
- `card_nav.html.twig` to `card_navigation.html.twig`
- `card_nav_header.html.twig` to `card_navigation_header.html.twig`

What is the second question?

**Comment by cavasinf** [comment]
It was yours:
> Is the only reason the icon in your example?

And I respond:
>Not only, in my case I add sometimes badges/dropdown/close button/...
So I need to have the fully control of what I can put in there.

But if you merge the PR, it maybe means that I was right? 😄

**Pull request merged**